CHANGES.md

```diff
@@ -6,6 +6,7 @@ wavesurfer.js changelog
 - Regions plugin:
   - Improved delta calculation (resize end) (#2641)
 - Fix iphone silent switch webaudio mute (#2667)
+- Add remaining mediaelement events (#2691, #2694)
```

Comment:
Please remove `remaining`, it's meaningless to non-contributors. Adding a list or link to these  specific events would be preferable.

Reply:
updated to make it more descriptive